whipper/command/cd.py

```diff
@@ -341,7 +346,10 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = self.program.getCoverArt(dirname,
```

Comment:
I think the check for the missing (optional) `pillow` dependency should be put here. Maybe something like this:

```python
self.coverArtPath = None
try:
    from PIL import Image
    self.coverArtPath = self.program.getCoverArt(dirname,
                                                 self.options.cover_art,
                                                 self.program.metadata.mbid)
except ImportError as e:
    logger.warning("the cover art option won't be honored because "
                   "the 'pillow' module isn't available: %r", e)
```

Reply:
This avoid importing the module and should be free of warnings ;)

```python
        self.coverArtPath = None
        import importlib.util
        if importlib.util.find_spec("PIL.Image") is not None:
            self.coverArtPath = self.program.getCoverArt(
                                    dirname,
                                    self.options.cover_art,
                                    self.program.metadata.mbid)
        else:
            logger.warning("the cover art option won't be honored because "
                           "the 'pillow' module isn't available: %r", e)
```

Reply:
Sorry, the `e` variable isn't defined in the code above: just drop it:

```python
            logger.warning("the cover art option won't be honored because "
                           "the 'pillow' module isn't available")
```